project_euler/problem_56/sol1.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import print_function
```

Comment:
We don't need this anymore. As stated in our contribution guide, Python means Python 3.x+.

Reply:
https://docs.python.org/3/howto/pyporting.html#prevent-compatibility-regressions

This directive changes how Python 2 operates.  It makes it a syntax error in Python 2 to add a legacy __print__ statement.  This prevents Python 2 developers from breaking Python 3 compatibility.

Reply:
Right. If we consider any forward compatibility, this should be here. It doesn't matter a lot as long as it's on the first line.